lte/gateway/python/integ_tests/s1aptests/test_attach_ipv4v6_pdn_type.py

```diff
@@ -28,19 +28,16 @@ def test_attach_ipv4v6_pdn_type(self):
         """ Test Attach for the UEs that are dual IP stack IPv4v6
             capable """
         # Set PDN TYPE to IPv4V6 i.e. 3. IPV4 is equal to 1
-        # IPV4orIPv6 is equal to 3 in value
         resp_ipv4oripv6 = self._create_attach_ipv4v6_pdn_type_req(
```

Comment:
why is this variable named `resp_ipv4oripv6`? should not it be `resp_ipv4andipv6`?

Reply:
This is an old testcase and the variables are named as per s6a spec 29.272 in which both ipv4andv6 and ipv4orv6 are defined. However while requesting a PDN type UE should only use ipv4, ipv6 or ipv4v6. I will change the variable names